src/lib/components/search/display/images/Images.svelte

```diff
@@ -22,31 +22,27 @@
 			<Preview result={imgResultPreview} />
 		</div>
 	{/if}
-	<section
-		id="images"
-		class:tw-w-2-3={imgResultPreview !== undefined}
-		class="flex flex-wrap justify-center gap-2"
-	>
-		{#each results as result (result.URL)}
-			<div class="flex-none">
-				<Image {result} bind:imgResultPreview />
-			</div>
-		{/each}
-	</section>
+	<div class="w-full">
+		<section id="images" class="grid grid-cols-fit auto-rows-[200px] grid-flow-dense gap-2">
+			{#each results as result (result.URL)}
+				<div
+					class:row-span-2={(result.ImageResult.Original.Width * 0.8) /
+						result.ImageResult.Original.Height >
```

Comment:
Switch to using thumbnail for calculating aspect ratio, since that is what is being used as source in the grid.

Reply:
As I've said, aspect ratio is a _proportional relationship_.
I'll change it so that the code-base is more consistent.
Google Images issue that we're dealing with is not related to this, I'm certain.

Reply:
_declared as a bug which was in turn solved so this can be resolved_